tests/test_sink_and_static.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+import falcon
+from falcon import testing
+
+from _util import create_app  # NOQA
+
+
+def sink(req, res, **kw):
+    res.body = 'sink'
+
+
+async def sink_async(req, res, **kw):
```

Comment:
It would be probably more idiomatic / falconic to use `req` and `resp` as everywhere else.